apps/web/src/modules/job/list/pages/types.ts

```diff
@@ -15,6 +15,8 @@ export interface IFilter {
   maleCheck: boolean
   femaleCheck: boolean
   otherCheck: boolean
+  cancleCheck: boolean
```

Comment:
Please change all object key and types which related to this also
```suggestion
  cancelCheck: boolean
```

Reply:
ตาโคตรดี :joy: